src/renderer/services/file-management-system/index.ts

```diff
@@ -86,11 +86,50 @@ export default class FileManagementSystem {
       serviceFields: {
         files: [metadata],
         type: "upload",
+        // localNasShortcut: true, // TODO for reliable pilot testing
```

Comment:
this line seems deletable

Reply:
im uncommenting that for pilot testing on windows.